app/src/main/java/com/example/cardgame/GameplayFragment.kt

```diff
@@ -335,69 +217,105 @@ class GameplayFragment : Fragment() {
-                    val image = strRes.cards[0].image
-                    val cardValue = strRes.cards[0].value
-                    returnCard = Consts.indexOf(cardValue)
                     activity?.runOnUiThread {
-                        val imgView = root.findViewById<ImageView>(R.id.ivCurrentCard)
+
+                        val imgBack = root!!.findViewById<ImageView>(R.id.ivCurrentCardBack)
+                        val imgView = root!!.findViewById<ImageView>(R.id.ivCurrentCard)
+                        val easyFlipView = root!!.findViewById<EasyFlipView>(R.id.easyFlipView)
+
+                        Picasso.get().load(R.drawable.back_of_a_card).into(imgBack)
+
+                        easyFlipView.flipDuration = 500
+                        easyFlipView.setFlipTypeFromLeft()
+                        easyFlipView.flipTheView(true)
+
+                        easyFlipView.flipDuration = 2000
+                        easyFlipView.setFlipTypeFromLeft()
+                        easyFlipView.flipTheView(true)
+
                         Picasso.get().load(image).into(imgView)
+
+                        val tv = root!!.findViewById<TextView>(R.id.tvScore)
+                        tv.text = currentScore.toString()
                     }
-                }
-                else {
-                    Log.d(TAG, "RESPONSE FOR NEW CARD IS UNSUCCESSFUL")
+
+                    val nextCard = returnCard
+                    if (shouldCompare) {
+                        if (op(nextCard, currentCard)) {
+                            if (currentCard > nextCard)
+                                Log.d(
+                                    TAG,
+                                    "${Consts.getName(nextCard)} < ${Consts.getName(currentCard)}"
+                                )
+                            else
+                                Log.d(
+                                    TAG,
+                                    "${Consts.getName(nextCard)} > ${Consts.getName(currentCard)}"
+                                )
+
+                            currentCard = nextCard
+                            currentScore++
+                            activity?.runOnUiThread {
+                                val tvScore = root!!.findViewById<TextView>(R.id.tvScore)
+                                tvScore.text = currentScore.toString()
+                            }
+                        } else {
+                            if (currentCard > nextCard)
+                                Log.d(
+                                    TAG,
+                                    "${Consts.getName(nextCard)} < ${Consts.getName(currentCard)}"
+                                )
+                            else
+                                Log.d(
+                                    TAG,
+                                    "${Consts.getName(nextCard)} > ${Consts.getName(currentCard)}"
+                                )
```

Comment:
This can then be removed since the values will be logged when evaluating if statement.